tiatoolbox/data/pretrained_model.yaml

```diff
@@ -771,3 +771,27 @@ micronet-consep:
             patch_output_shape: [252, 252]
             stride_shape: [150, 150]
             save_resolution: {'units': 'mpp', 'resolution': 0.25}
+
+sccnn-crchisto:
```

Comment:
consider adding input and output resolution as well.

Reply:
Will do while adding the engine